src/nvim/getchar.c

```diff
@@ -144,7 +144,7 @@ static int KeyNoremap = 0;                  /* remapping flags */
 static char_u typebuf_init[TYPELEN_INIT];       /* initial typebuf.tb_buf */
 static char_u noremapbuf_init[TYPELEN_INIT];    /* initial typebuf.tb_noremap */
 
-static int last_recorded_len = 0;       /* number of last recorded chars */
+static unsigned int last_recorded_len = 0;      // number of last recorded chars
```

Comment:
in general, if one can interpret the number as a memory size, it is better to use `size_t` than `unsigned int`.